### ISSUE TITLE
kwic keywords not vectorised

### ISSUE THREAD

**Issue by kbenoit** [kbenoit/quanteda#306]
The manual page suggests they are, but they are not:

```r
mycorpus <- corpus(c("This is a test",
                     "This is it.",
                     "What is in a train?",
                     "Is it a question?",
                     "Sometimes you don't know if this is it.",
                     "Is it a bird or a plane or is it a train?"))

kwic(mycorpus, c("is", "a"), valuetype = "fixed")
##              contextPre keyword contextPost
## [text1, 2:3]     This [    is a          ] 
```
This should match all occurrences of `is` and of `a` separately, but appears to concatenate them to form a sequence matching only the `is a` in `text1`.

Thanks to @jjphorn for bringing this to my attention.

**Comment by koheiw** [comment]
kwic2() in dev_kwic outputs like this:

```
> kwic2(mycorpus, c("is", "a"), valuetype = "fixed")
                        before keyword              after
text1:1                   This  [is a] test              
text2:1                   This    [is] it .              
text3:1                   What    [is] in a train ?      
text3:2             What is in     [a] train ?           
text4:1                           [Is] it a question ?   
text4:2                  Is it     [a] question ?        
text5:1 you don't know if this    [is] it .              
text6:1                           [Is] it a bird or a    
text6:2                  Is it     [a] bird or a plane or
text6:3        Is it a bird or     [a] plane or is it a  
text6:4     bird or a plane or    [is] it a train ?      
text6:5       a plane or is it     [a] train ?           
> kwic2(mycorpus, list("is", "a", c("is", "it")), valuetype = "fixed")
                        before   keyword              after
text1:1                   This    [is a] test              
text2:1                   This   [is it] .                 
text3:1                   What      [is] in a train ?      
text3:2             What is in       [a] train ?           
text4:1                        [Is it a] question ?        
text5:1 you don't know if this   [is it] .                 
text6:1                        [Is it a] bird or a plane or
text6:2        Is it a bird or       [a] plane or is it a  
text6:3     bird or a plane or [is it a] train ? 
```

**Comment by koheiw** [comment]
@kbenoit, if you think this is close to how it should be. Please take over the branch.

**Comment by kbenoit** [comment]
OK, will do. Did you redefine the print method? The bracket in `print.kwic` is appended to the before and after parts, not to the keyword, in `print.kwic`.[](url)

**Comment by koheiw** [comment]
I added the print method for `kwic2`.

**Comment by kbenoit** [comment]
Solved in 2075c21
